docs/using.md

```diff
@@ -9,11 +9,12 @@ overview of different components and services.
   - [Starting OpenShift](#starting-openshift)
   - [Stopping OpenShift](#stopping-openshift)
   - [Deleting OpenShift](#deleting-openshift)
-- [Environment variables](#environment-variables)
-- [Persistent configuration](#persistent-configuration)
-  - [Configuration options precedence](#configuration-options-precedence)
-  - [Setting persistent configuration values](#setting-persistent-configuration-values)
-  - [Unsetting persistent configuration values](#unsetting-persistent-configuration-values)
+- [Controlling runtime behavior of Minishift](controlling-runtime-behavior-of-minishift)
```

Comment:
Controlling runtime options, implies that we are controlling the options. So, are we controlling Minishft's behavior through these options or are we controlling the options themselves? AFAIU we customize the options in order to control Minshift's runtime behavior, no?
How about 'Minishift runtime options' ?